target/linux/lantiq/files-4.14/arch/mips/boot/dts/FRITZ7362SL.dts

```diff
@@ -0,0 +1,285 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7362sl", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7362 SL";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &power_red;
+		led-running = &power_green;
+		led-upgrade = &power_green;
+
+		led-dsl = &info_green;
+		led-wifi = &wifi;
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	gpio-keys-polled {
```

Comment:
@danijelt  Could you please change this to generic name `keys` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist)
>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation

Reply:
Done.